experimental/cloudstrapper/playbooks/roles/build-platform/files/python-configure.bash

```diff
@@ -0,0 +1 @@
+sudo apt-get install python3.7
```

Comment:
Why don't we do this via ansible's "Apt" itself

Reply:
Also is this being used?

Reply:
Vestiges of an earlier draft. Removed.